packages/schema/bind/src/bindings/golang/wasm/templates/imported/object-type/Object%type%Serialization-go.mustache

```diff
@@ -15,8 +15,8 @@ func serialize{{#toUpper}}{{type}}{{/toUpper}}(value *{{#toUpper}}{{type}}{{/toU
 func write{{#toUpper}}{{type}}{{/toUpper}}(writer msgpack.Write, value *{{#toUpper}}{{type}}{{/toUpper}}) {
 	writer.WriteMapLength({{properties.length}})
 	{{#properties}}
-	writer.Context().Push("{{#toUpper}}{{#handleKeywords}}{{name}}{{/handleKeywords}}{{/toUpper}}", "{{#toWasm}}{{toGraphQLType}}{{/toWasm}}", "writing property")
-	writer.WriteString("{{#toUpper}}{{#handleKeywords}}{{name}}{{/handleKeywords}}{{/toUpper}}")
+	writer.Context().Push("{{#toLower}}{{#handleKeywords}}{{name}}{{/handleKeywords}}{{/toLower}}", "{{#toWasm}}{{toGraphQLType}}{{/toWasm}}", "writing property")
+	writer.WriteString("{{#toLower}}{{#handleKeywords}}{{name}}{{/handleKeywords}}{{/toLower}}")
```

Comment:
This is incorrect, the name should be 1:1 with what's in the schema, so just {{name}}.